### PR TITLE
Make weighted_cross_entropy_with_logits consistent with sigmoid_cross_entropy_with_logits

### LINKED ISSUE #26337: Make weighted_cross_entropy_with_logits consistent with sigmoid_cross_entropy_with_logits in signature
**System information**
- TensorFlow version (you are using): 1.13
- Are you willing to contribute it (Yes/No): No

**Describe the feature and the current behavior/state.**
Currently, the signature of the two functions are:
```python
@tf_export("nn.sigmoid_cross_entropy_with_logits")
def sigmoid_cross_entropy_with_logits(  # pylint: disable=invalid-name
    _sentinel=None,
    labels=None,
    logits=None,
    name=None):
```
And 
```python
@tf_export("nn.weighted_cross_entropy_with_logits")
def weighted_cross_entropy_with_logits(targets, logits, pos_weight, name=None):
```

TF2.0 is a good chance to make them consistent in the naming.

**Will this change the current api? How?**
Yes.

**Who will benefit with this feature?**
Users of the two functions will be less confused.

**Any Other info.**
A follow up of #7086

### PR DESCRIPTION
**Pull request by yongtang**
This fix tries to address the issue raised in #26337 where weighted_cross_entropy_with_logits and sigmoid_cross_entropy_with_logits are not consistent with funtion definitions.

This fix changes the definitions in v2 so that they are consistent, but keep the old defs in v1.

Note this PR also fixes sparse_softmax_cross_entropy_with_logits consistency issue, as was specified in #26337.

This fix fixes #26337.

Signed-off-by: Yong Tang <yong.tang.github@outlook.com>

### CODE REVIEW

File: tensorflow/tools/compatibility/tf_upgrade_v2.py [hunk 1]
```diff
@@ -440,6 +440,12 @@ def __init__(self):
             "tensor": "data",
             "family": None,
         },
+        "tf.nn.weighted_cross_entropy_with_logits": {
+            "targets": "labels",
+        },
+        "tf.nn.sparse_softmax_cross_entropy_with_logits": {
```
Comment (alextp) [inline tensorflow/tools/compatibility/tf_upgrade_v2.py]: I don't think we need _sentinel here since no one is allowed to actually pass anything as the _sentinel

File: tensorflow/python/ops/nn_ops.py [hunk 2]
```diff
@@ -3363,6 +3363,61 @@ def sparse_softmax_cross_entropy_with_logits(
       else:
         return cost
 
+@tf_export("nn.sparse_softmax_cross_entropy_with_logits", v1=[])
+def sparse_softmax_cross_entropy_with_logits_v2(
+    labels=None,
```
Comment (ppwwyyxx) [inline tensorflow/python/ops/nn_ops.py]: The `None` defaults for `labels` and `logits` should be removed, because these two arguments are always required.
They are needed in v1, however, because v1 uses sentinels.

Comment (yongtang) [reply tensorflow/python/ops/nn_ops.py]: @ppwwyyxx Thanks. The PR has been updated.

**Review by alextp — APPROVED** [review]
